ci_framework/roles/artifacts/tasks/edpm.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: List all of the existing virtual machines
+  become: true
```

Comment:
I don't think "become: true" is needed here - we're using the qemu:///system uri.

Reply:
Done